test/unit/spec/common/cellComponents/fieldTableCellWidget-Spec.js

```diff
@@ -85,9 +85,11 @@ define([
             Jupyter.narrative = null;
         });
 
-        beforeEach(async () => {
+        beforeEach(async function () {
+            this.parent = document.createElement('div');
```

Comment:
this widget uses the parent of the node that it is supplied with, so create a parent node and then delete that when all the tests have finished. That should minimise the risk of deleting the whole document body and messing up another test that might be running at the same time.